trunk/includes/pardot-settings-class.php

```diff
@@ -286,30 +273,6 @@ function admin_head() {
 <script>
 jQuery(document).ready(function(){jQuery("#campaign").chosen();});
 
-jQuery(document).ready(function($){
-    $('#auth-type').change(function() {      
-        if (this.value === 'pardot') {        
-            $('#email-wrap').parents().eq(1).show();
-            $('#password-wrap').parents().eq(1).show();  
-            $('#user-key-wrap').parents().eq(1).show();
-            $('#client-id-wrap').parents().eq(1).hide();
-            $('#client-secret-wrap').parents().eq(1).hide(); 
-            $('#business-unit-id-wrap').parents().eq(1).hide();
-            $('#sso-sign-in').hide();
-            
-        } else if (this.value === 'sso') {        
-            $('#email-wrap').parents().eq(1).hide();    
-            $('#password-wrap').parents().eq(1).hide();
-            $('#user-key-wrap').parents().eq(1).hide();
-            $('#client-id-wrap').parents().eq(1).show();
-            $('#client-secret-wrap').parents().eq(1).show(); 
-            $('#business-unit-id-wrap').parents().eq(1).show();
-            $('#sso-sign-in').show();
-        }
-    });
-
-});
-
```

Comment:
Now unnecessary logic that would swap between showing the Pardot and SSO fields in the Pardot plugin settings page.